modules/microshift-gitops-limitations.adoc

```diff
@@ -0,0 +1,16 @@
+
+// Module included in the following assemblies:
+//
+// microshift_running_apps/microshift-gitops.adoc
+
+:_mod-docs-content-type: CONCEPT
+[id="microshift-gitops-limitations_{context}"]
+= Limitations of using the GitOps agent with {microshift-short}
+
+[role="_abstract"]
+Using GitOps with Argo CD for {microshift-short} is different from using the entire {gitops-title} Operator in the following ways:
```

Comment:
s/b _different than_, but who cares?  I've been trying to learn different from/than forever! 
See: https://redhat-documentation.github.io/supplementary-style-guide/#different

Reply:
Super weird, because I get a Vale error that says to use "from." 
<img width="757" height="195" alt="Screenshot 2025-11-11 at 7 04 07 AM" src="https://github.com/user-attachments/assets/4d9e03d1-3334-414c-a262-fa70ccbafdeb" />